usaspending_api/search/filters/elasticsearch/HierarchicalFilter.py

```diff
@@ -17,7 +17,7 @@ def _query_string(cls, require, exclude) -> str:
         negative_query = " AND ".join([node.get_query() for node in negative_nodes])
 
         if positive_query and negative_query:
-            return f"{positive_query} AND {negative_query}"
+            return f"({positive_query}) AND {negative_query}"
```

Comment:
Likely to conflict with https://github.com/fedspendingtransparency/usaspending-api/pull/2465/files#diff-99d9fa9cbd9aa0439c711d228cffb43cR20, but if you need this for your tests to work it might be worth swallowing that conflict.

Reply:
Yeah.  First in wins?  Second in reconciles?